pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -1090,8 +1090,6 @@ def test_setitem_partial_column_inplace(self, consolidate, using_array_manager):
         # check setting occurred in-place
         tm.assert_numpy_array_equal(zvals, expected.values)
         assert np.shares_memory(zvals, df["z"]._values)
-        if not consolidate:
-            assert df["z"]._values is zvals
```

Comment:
See https://github.com/pandas-dev/pandas/pull/47074#discussion_r879227685 for comment about this removal